OpenBB-finance/OpenBBTerminal
# Switches all multi input arguments to comma separation

# Description

Fixes #2887 
Fixes #2919 

This PR standardizes multi argument calls in the economy and etf menu. 

Note: The econometrics menu did not need any changes
Note2: There are still a lot of `nargs="+"` lines left. This is an easy way of allowing multi word arguments (like 'United States' for a country argument).
Note 2.5: I could not find a good way to send multiple arguments to the `stocks/fa` commands: `cash`, `balance`, and `income` because each individual argument can be one ore multiple words.
Note3: Also fixes the following bugs:

- `etf/pir` would throw an error if `OpenBBUserData/exports/etf` did not already exist
- "[/red" was displayed when the main `news` command has a timeout error

# How has this been tested?
This only affects controlers and sending arguments. So reports and the API should not be affected. 

- [x] Make sure affected commands still run in terminal
    - `python terminal.py economy/macro "-p CPI,PPI -c canada,germany"`
    - `python terminal.py economy/fred "-p CPI,PPI -q stocks,bonds"`
    - `python terminal.py "economy/index -i nasdaq,sp500 -q oil,"`
    - `python terminal.py "economy/treasury -m 10y,20y -t nominal,inflation"`
    - `plot --y1 Nominal_10-year,Nominal_20-year --y2 Inflation_10-year,Inflation_20-year` (must run previous command first)
    - `python terminal.py "etf/load qqq/news -s bbc,yahoo.com"`
    - `python terminal.py "etf/pir -e QQQ,VOO"`
    - `python terminal.py "news -s=bloomberg.com,yahoo.com"`

    To test country refactor:
    - `python terminal.py "economy/ycrv -c united_kingdom --source Investing" --debug`
    - To see the new country recommendations are correct: `terminal.py ycrv -c united_kingdo --source Investing` (notice the underscore for multi word countries)
    - Spread was fixed but not tested because it was taken off the menu.
    - `python terminal.py "alternative/covid/country -c united_kingdom"`
    - `python terminal.py "stocks/search -c united_states"`
    - To see recommendations are correct: `python terminal.py "stocks/search -c united_state"` (parser should give a list of valid options)
- [x] Ensure the api still works
    - Just had to check the endpoints that were affected by my model and view changes:
    - `from openbb_terminal.sdk import openbb`
    - `openbb.economy.index("nasdaq")`
    - `openbb.common.news()`
- [x] Check any related reports
    - A lot of reports are not working for me right now. This is fixed by #2878 


# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).


# Others
